pkg/commands/process/settings/rules.go

```diff
@@ -17,6 +17,14 @@ var (
 	defaultAuxiliaryRuleType = "verifier"
 )
 
+func GetSupportedRuleLanguages() map[string]bool {
+	return map[string]bool{
+		"ruby":       true,
+		"javascript": true,
+		"typescript": true,
+	}
+}
```

Comment:
Hard coding this for speed: I'm wanting to know which languages have rule support without doing any API calls e.g. to get the folder names from bearer-rules. 

Not sure this solution is ideal - am open to suggestions 🙏